core/tasks.py

```diff
@@ -369,8 +369,7 @@ def execute(self, taskID, *args, **kwargs):
             raise errors.Unauthorized()
         skel = task.dataSkel()
         skey = kwargs.get("skey", "")
-        if len(kwargs) == 0 or skey == "" or not skel.fromClient(kwargs) or (
-            "bounce" in kwargs and kwargs["bounce"] == "1"):
+        if len(kwargs) == 0 or not skel.fromClient(kwargs) or kwargs.get("bounce") == "1":
```

Comment:
Generally, I would like to propose here to test the "bounce"-parameter on `conf["viur.bone.boolean.str2true"]` but this should be done in a separate PR.